tensorboard/webapp/metrics/store/metrics_types.ts

```diff
@@ -126,10 +126,18 @@ export type CardMetadataMap = Record<
   CardMetadata
 >;
 
+export enum CardSelectionState {
+  GLOBAL,
```

Comment:
I'm not in love with the name `GLOBAL` but I want to convey the intention to not provide an override.